### PR TITLE
chore: sign up activity cancel dialog box updated

### PR DESCRIPTION
**Pull request by naman653**
Changes: On pressing back button at sign up activity, a dialog box appears for confirmation which is a bit confusing. You can see in the below image that both options can be thought as the option to exit the activity by different users. So I've updated the text so as to provide convinient options.

Before:
screenshot_2019-02-28-19-17-31

After:
screenshot_2019-02-28-19-18-43

### CODE REVIEW

**Comment by naman653** [comment]
@iamareebjamal sir, please review

**Comment by iamareebjamal** [comment]
I think it is fine as it is

**Comment by naman653** [comment]
I personally got confused many times. Then I experimented on my friend and he got confused also. If you think so, then please review it last time, otherwise happy to close this PR.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -204,7 +204,7 @@
     <string name="invalid_credentials">Invalid credentials. Please try again</string>
 
     <!-- Settings Activity-->
-    <string name="Continue">Continue</string>
```
Comment (iamareebjamal) [inline app/src/main/res/values/strings.xml]: Don't remove this

**Comment by naman653** [comment]
@batbrain7 @iamareebjamal @arundhati24 Please merge this PR.

**Pull request merged**